src/trajectory/Serialize.java

```diff
@@ -0,0 +1,28 @@
+package trajectory;
+
```

Comment:
In general, you want to avoid `wildcard` or `*` imports. Especially so because this is getting deployed on a robot!

```
import java.io.BufferedReader;
import java.io.FileReader;
```